src/libsystemd/sd-bus/bus-objects.c

```diff
@@ -1610,6 +1610,39 @@ static int vtable_member_compare_func(const struct vtable_member *x, const struc
 
 DEFINE_PRIVATE_HASH_OPS(vtable_member_hash_ops, struct vtable_member, vtable_member_hash_func, vtable_member_compare_func);
 
+static bool names_are_valid(const char *signature, const char **names, int *flags) {
+        enum {
+                first           = 1<<0,
```

Comment:
we usually uppercase the names used in enums, please do so here too.

Also we usually (but not always unfortunately) put a space before and after each operator, i.e. please write `1 << 0`